desktop/core/src/desktop/lib/django_util.py

```diff
@@ -468,10 +468,13 @@ class JsonResponse(HttpResponse):
       to ``True``.
```

Comment:
Could you add the comment?

    :param json_dumps_params: A dictionary of kwargs passed to json.dumps().